content/docs/console/integrations/webhooks.md

```diff
@@ -55,6 +57,7 @@ Organization-level webhooks will be sent webhook events from all of the stacks w
 that organization.
 
 | Event Kind | Triggered |
+| --- | --- |
```

Comment:
I noticed the tables in the rest of this page aren't being rendered correctly, so perhaps a similar change is needed for the tables below.

Reply:
Yeah, thanks. Fixed.